runtime/runtime_controller.cc

```diff
@@ -50,11 +50,12 @@ std::unique_ptr<RuntimeController> RuntimeController::Spawn(
     const fml::closure& p_isolate_shutdown_callback,
     std::shared_ptr<const fml::Mapping> p_persistent_isolate_data,
     fml::WeakPtr<IOManager> io_manager,
-    fml::WeakPtr<ImageDecoder> image_decoder) const {
+    fml::WeakPtr<ImageDecoder> image_decoder,
+    fml::WeakPtr<ImageGeneratorRegistry> image_generator_registry) const {
```

Comment:
It would be nice to clean this up at some point by eliminating parameters.  We are almost passing in a full UIDartState::Context.  I looked into it briefly and it wasn't trivial.